geode-core/src/main/java/org/apache/geode/internal/cache/AbstractBucketRegionQueue.java

```diff
@@ -223,7 +231,12 @@ protected void loadEventsFromTempQueue() {
       // .getBucketToTempQueueMap().get(getId());
       if (tempQueue != null && !tempQueue.isEmpty()) {
         synchronized (tempQueue) {
+          Map<String, Map<Integer, List<Object>>> regionToDuplicateEventsMap =
+              new ConcurrentHashMap<>();
           try {
+            boolean notifyDuplicate =
```

Comment:
Extract to a method, say `boolean notifyDuplicateSupported()`, for readability.

Reply:
Thanks for comments. Updated.